homeassistant/components/iron_os/strings.json

```diff
@@ -79,6 +83,81 @@
         "name": "Long-press temperature step"
       }
     },
+    "select": {
+      "min_dc_voltage_cells": {
+        "name": "Power source",
+        "state": {
+          "no_battery": "External power supply (DC)",
```

Comment:
What happens if a user changes this setting to the wrong value?

Reply:
If used with a power supply, nothing. If used with a power bank that has proper under-voltage protection, also nothing, except that the iron may shut off earlier than necessary. If a battery without under-voltage protection is used, it may be harmful and reduce its lifespan. But regardless, it is in any case not recommended to fully drain a LI-Ion battery 🤷🏼

Reply:
In the documentation I also noted, that advanced settings are deactivated by default, I consider this as an advanced setting, although on the settings menu on the device it can be found under power settings and not advanced settings.